lighthouse-viewer/test/psi-api-test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const psiApi = require('../app/src/psi-api.js');
+const Config = require('../../lighthouse-core/config/config.js');
+const defaultConfig = require('../../lighthouse-core/config/default-config.js');
+
+describe('PSI API', () => {
+  it('default psi categories is same as default config categories', () => {
+    const categories = Config.getCategories(defaultConfig).map(c => c.id).sort();
+    expect(psiApi.PSI_DEFAULT_CATEGORIES.slice(0).sort()).toEqual(categories);
```

Comment:
> seems like this might be better in the integration test? I also see the appeal having it in here, but we also don't really care about the internal representation, just what comes out in the API request (and since we already mock the request in the integration test, might be easier there rather than doing the same in a different way in here).

and actually, the `should call out to PSI with all categories by default` in `viewer-tst-pptr.js` is already doing most of this work, maybe just check it against `lighthouseCategories` (which is already pulled from the default config) instead of an inline literal and add a comment that that's also being checked

Reply:
the inline literal is necessary as the _order_ is very important for the PSI API, but yes, this test could be moved to the intergration test.